addons/mail/static/tests/qunit_suite_tests/components/emojis_field_common_tests.js

```diff
@@ -0,0 +1,61 @@
+/**@odoo-module **/
+
+import { click } from "@web/../tests/helpers/utils";
+
+/**
+ * Check that the emoji button is not visible
+ *
+ * @param {object} assert the assert object passed by QUnit
+ * @param {HTMLElement} target fixture where the view is displayed
+ * @param {string} emojiComponentSelector unique selector to get the component template root (e.g. ".o_field_text_emojis")
+ */
+export async function testEmojiButtonHidden(assert, target, emojiComponentSelector) {
+    assert.containsOnce(target, ".o_form_readonly", "Form should be readonly for this test");
+    assert.containsOnce(target, emojiComponentSelector, "The field component is not in the dom");
```

Comment:
You do "containsOnce" here and the comment says "is NOT in the dom".

A bit confusing ^^

Reply:
I guess these are all pretty self-explanatory so I'll just remove the custom messages. It is confusing indeed ^^